src/pages/Dashboard.tsx

```diff
@@ -214,4 +237,105 @@ const Dashboard = () => {
   );
 };
 
+const MembershipStatus = ({
```

Comment:
Definitely extract this to a separate component, there is a `components/dashboard` folder where you can add it

Reply:
If you do this, can also add a story :)